### ISSUE TITLE
sub sublevels?

### ISSUE THREAD

**Issue by pfraze** [dominictarr/mynosql#4]
Hey, I'm wondering if using sublevels (log, meta) directly off the received db could result sublevel collisions. SSB uses a log sublevel.

Could they go in a 'mynosql' sublevel? `db.sublevel('mynosql').sublevel('log')`?

**Comment by juliangruber** [comment]
you should be able to pass `db.sublevel('mynosql')` to this module, to avoid conflicts

**Comment by pfraze** [comment]
@juliangruber if you did that, would the indexing in https://github.com/dominictarr/mynosql/blob/master/index.js#L40 catch everything on the db, or would it only catch data added to the mynosql sublevel?

**Comment by pfraze** [comment]
hm, the proposed merge of mynosql with ssb puts the `log` sublevel into mynosql's hands https://github.com/ssbc/secure-scuttlebutt/pull/95/files#diff-168726dbe96b3ce427e7fedce31bb0bcR96

**Comment by dominictarr** [comment]
@pfraze yes I `log` sublevels use contain exactly the same data in these cases.

**Comment by pfraze** [comment]
ok, that's fine then. Might want to document the sublevel use in the readme

**Comment by dominictarr** [comment]
@pfraze the mynosql readme?

**Comment by pfraze** [comment]
@dominictarr yeah

**Comment by dominictarr** [comment]
fixed in 2.2.1

**Comment by pfraze** [comment]
:+1:

**Comment by NHQ** [comment]
BTW level-sublevel was not added to the pkg.json
